### PR TITLE
Don't use multiline pairs for head/tail modifier.

### PR DESCRIPTION
**Pull request by AndreasArvidsson**
`change tail` did the wrong thing in this case
```py
"""|foo
 bar
"""
```

## Checklist

- [x] I have added [tests](https://www.cursorless.org/docs/contributing/test-case-recorder/)
- [/] I have updated the [docs](https://github.com/cursorless-dev/cursorless/tree/main/docs) and [cheatsheet](https://github.com/cursorless-dev/cursorless/tree/main/cursorless-talon/src/cheatsheet)
- [/] I have not broken the cheatsheet

### CODE REVIEW

**Comment by AndreasArvidsson** [comment]
@saidelike Looks like our neovim lua tests are failing
https://github.com/cursorless-dev/cursorless/blob/49f70840f0b11d4fca1744f88af1cd3d91cbd3d5/.github/actions/test-neovim-lua/action.yml

Here is a suggested solution. Do you want to look at this?
https://github.com/leafo/gh-actions-lua/issues/49

File: packages/cursorless-engine/src/processTargets/modifiers/scopeHandlers/SurroundingPairScopeHandler/SurroundingPairInteriorScopeHandler.ts [hunk 1]
```diff
@@ -32,28 +31,31 @@ export class SurroundingPairInteriorScopeHandler extends BaseScopeHandler {
...
-      }),
+    const scopes = this.surroundingPairScopeHandler.generateScopes(
+      editor,
+      position,
+      direction,
+      hints,
     );
+
+    for (const scope of scopes) {
+      if (!this.scopeType.requireSingleLine || scope.domain.isSingleLine) {
```
Comment (phillco) [inline packages/cursorless-engine/src/processTargets/modifiers/scopeHandlers/SurroundingPairScopeHandler/SurroundingPairInteriorScopeHandler.ts:48]: This reads a touch easier to me.
```suggestion
      if (scope.domain.isSingleLine || !this.scopeType.requireSingleLine) {
```

**Review by phillco — APPROVED** [review]
Thanks, sorry I missed this.

File: packages/cursorless-engine/src/processTargets/modifiers/scopeHandlers/SurroundingPairScopeHandler/SurroundingPairInteriorScopeHandler.ts (hunk 1 above)
Comment (AndreasArvidsson) [reply packages/cursorless-engine/src/processTargets/modifiers/scopeHandlers/SurroundingPairScopeHandler/SurroundingPairInteriorScopeHandler.ts:48]: I think the intent is clearer the way I wrote it, but it's not to important either way.

**Pull request merged**